packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/theme/design_system/design_system_colors.dart

```diff
@@ -73,6 +73,8 @@ class DesignSystemColors {
 
   final gray = const Color(0xff808080);
 
+  final lightGray = const Color(0xFFE1E1E1);
```

Comment:
Please do not specify the actual color in the property name, since it will differ in dark mode

@JvnSlv FYI